docs/src/components/pricing/FAQ.tsx

```diff
@@ -63,6 +58,34 @@ const faqData = [
       </React.Fragment>
     ),
   },
+  {
+    summary: 'Why do we must license developers not using the software directly?',
+    detail: (
+      <React.Fragment>
+        Our pricing model requires all developers working on a project using MUI X Pro or Premium to
+        be licensed. This is designed to make it easier for your engineers to use the software
+        without having to constantly verify if they have the right number of seats. It aims to
```

Comment:
```suggestion
        without having to verify whether they have the correct number of seats.

Reply:
I have reworked the content, changing the meaning a bit. It's not so much about verifying but more about knowing.